code/modules/reagents/reagent_dispenser.dm

```diff
@@ -168,7 +168,7 @@
 
 /obj/structure/reagent_dispensers/fueltank/bullet_act(var/obj/item/projectile/Proj)
 	if(istype(Proj ,/obj/item/projectile/beam)||istype(Proj,/obj/item/projectile/bullet)||istype(Proj,/obj/item/projectile/ricochet))
-		if(!istype(Proj ,/obj/item/projectile/beam/lasertag) && !istype(Proj ,/obj/item/projectile/beam/practice) )
+		if(!istype(Proj ,/obj/item/projectile/beam/lasertag) && !istype(Proj ,/obj/item/projectile/beam/practice) && Proj.get_damage())
```

Comment:
shouldn't the rest of this ifcheck be redundant now?

Reply:
nope, the lasertag and practice laser guns deal "holo" damage which still counts as damage.

Reply:
maybe check to see if damage type is holo then

Reply:
nvm looks like you were right, no idea where I remember practice laser gun holo damage from then. I could swear someone lasered me with it years ago which made me fall into "fake crit" for a few seconds.